### PR TITLE
Add govuk template preview

### PR DESCRIPTION
**Pull request by gemmaleigh**
This PR adds a rails application, uses the govuk_template gem and applies the govuk_template layout, it is intended to be used to preview the govuk_template gem and also to [accompany the documentation here to get set up with Rails using the govuk_template](https://github.com/alphagov/govuk_template/blob/master/docs/using-with-rails.md).

cc. @dsingleton, @36degrees, @thehenster.

Comments welcome!

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -30,7 +30,7 @@ gem 'jbuilder', '~> 2.5'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
-gem 'govuk_template'
+gem 'govuk_template', '0.18.1'
```
Comment (thehenster) [inline Gemfile]: Yaknow you can run `bundle update govuk_template` which will find the latest compatible version?

Pinning to a version in the Gemfile stops the dependency manager from doing it's thing.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -14,10 +54,62 @@
...
+<% content_for :licence_message do %>
+<% end %>
+
+<%# Copyright message %>
+<%# default: © Crown copyright %>
+<% content_for :crown_copyright_message do %>
+<% end %>
+
+<%# Insertion point after closing </body> element %>
+<% content_for :body_end do %>
```
Comment (thehenster) [inline app/views/layouts/application.html.erb]: Do you think we need to worry about them going out of date as https://github.com/alphagov/govuk_template/blob/master/docs/template-blocks.md gets updated?

**Comment by gemmaleigh** [comment]
I spoke to @dsingleton about this, it would be better if this lived in the govuk_template repo, otherwise it is likely to get out-of-date.

**Comment by dsingleton** [comment]
> @dsingleton mentioned it would be better if this lived in the govuk_template repo, otherwise it is likely to get out-of-date.

👍 

The two things this repo does are really useful, but I think we can achieve them without the duplication/out-of-sync issues a second repo risks.

1. It acts as documentation for setting up a basic rails app. I think we should see how we get on with [just the docs](https://github.com/alphagov/govuk_template/blob/master/docs/using-with-rails.md) approach for now. We can see what we learn from research, and if there's a need for a second repo. If there is, we can think about how we keep things in sync, but for now I think we can side step it.
2. A way to preview changes to the template repo itself, which is not something we can do easily right now, but I think the ability to preview should probably live inside the `govuk_template` repo - something like a `bundle exec server` (or similar). That avoids the issues of duplication across repos, and having to deal with local file paths for development versions of the gem that haven't been published yet.

Both are really useful - and should definitely be part of the alpha work - but I think we can do it within the single repo, and that makes life easier in the long term  :)

**Pull request closed without merging**